res/controllers/Roland_DJ-202-scripts.js

```diff
@@ -62,6 +62,34 @@ DJ202.Deck = function (deckNumbers, offset) {
         },
     });
 
+    this.paramUp = function (channel, control, value, status, group) {
```

Comment:
Since the MIDI control is supplied, paramUp and paramDown could probably be merged into one binding.

Reply:
For now that's true. We'd probably have to rethink this in case we want to use them for layering the performance pads #5